rfc/rfc-0019-rfc-process.md

```diff
@@ -0,0 +1,112 @@
+
+# Motivation
+[motivation]: #motivation
+
+Now that we've started releasing minor versions and people are trying out Cartographer, we need to take more care in introducing change. This process documents a path for anyone to contribute and have input on features.
+
+# What it is
+[what-it-is]: #what-it-is
+
+RFCs record potential change with the context and information at the given time. This provides a defined process for anyone wishing to contribute to the Cartographer project and gives opportunity for engagement. Anyone who chooses not to actively partcipate in any RFC is presumed to trust their colleagues on the matter. Once an RFC is accepted, they can be referenced as read-only documents in this repository until replaced or ammended by another RFC when context has significantly changed.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the community and the maintainer team.
+
+## What's in Scope
+
+You'll need to follow this process for anything considered "substantial". The things that we consider "substantial" will evolve over time, but will generally be user facing changes.
+
+What constitutes a "substantial" change may include the following but is not limited to:
+
+- changes to a resource's spec
+- changes to a resource's status
+- breaking changes
+- governance changes
+- issues that need more discussion or clarification as determined by the project maintainers
+
+If you submit a pull request or issue that the team deems warrants an RFC, it will be politely closed with a request for an RFC.
+
+## Process
+
+### RFCs
+
+To get an RFC into Cartographer, first the RFC needs to be merged into the repo. Once an RFC is merged, it's considered 'active' and may be implemented to be included in the project. These steps will get an RFC to be considered:
+
+- Fork the repo: <https://github.com/vmware-tanzu/cartographer>
+- Copy `rfc/rfc-0000-template.md` to `rfc/rfc-0000-my-feature.md` (where 'my-feature' is descriptive. don't assign an RFC number yet).
+- Fill in RFC. Any section can be marked as "N/A" if not applicable.
+- Submit a pull request. The pull request is the time to get review of the proposal from the larger community.
+- Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+
+Once a pull request is opened, the RFC is now in development and the following will happen:
+
+- It will be discussed in a future community meeting. Community meetings happen on a weekly cadence barring exceptions.
```

Comment:
```suggestion
- It will be discussed in a future office hours meeting. Office hours happen on a weekly cadence barring exceptions.
```

David has hoped to keep RFCs to Office Hours